timescaledb/overview/data-model-flexibility/narrow-data-model.md

```diff
@@ -22,20 +23,23 @@ Using our example above, this approach would result in 9 different "time series"
 8. {name: temperature,  device_id: def456,  location_id: 335,  dev_type: roof}
 9. {name: temperature,  device_id: ghi789,  location_id:  77,  dev_type: roof}
 ```
-The number of such time series scales with the cross-product of the
-cardinality of each tag, i.e., (# names) &times; (# device ids) &times; (#
-location ids) &times; (device types). Some time-series databases struggle as
-cardinality increases, ultimately limiting the number of device types and devices
-you can store in a single database.
+The number of such time series scales with the cross-product of the cardinality
+of each tag, i.e., (# names) &times; (# device ids) &times; (# location ids)
+&times; (device types). Some time-series databases struggle as cardinality
+increases, ultimately limiting the number of device types and devices you can
+store in a single database.
```

Comment:
I don't understand what this means at all. Also, please remove the Latin abbreviation.